.github/workflows/pr-checks.yml

```diff
@@ -8,39 +8,39 @@ jobs:
     runs-on: ${{ matrix.os }}  # This is defined by the strategy below
     strategy:
       # The strategy allows to automatically test different os and python version combinations
-      fail-fast: true  # The PR checks should fail fast not to waste GitHub minutes
+      fail-fast: false  # The PR checks should fail fast not to waste GitHub minutes
```

Comment:
I just changed this to test all 5 versions on the PR. Will change back before merging.